1-js/02-first-steps/16-javascript-specials/article.md

```diff
@@ -248,40 +248,40 @@ We covered three ways to create a function in JavaScript:
     }
     ```
 
-    Function expressions can have a name, like `sum = function name(a, b)`, but that `name` is only visible inside that function.
+    Expressões de função podem ter um nome, como `sum = function name(a, b)`, mas esse `name` apenas é visível dentro da função.
 
 3. Arrow functions:
 
     ```js
-    // expression at the right side
+    // expressão no lado direito
     let sum = (a, b) => a + b;
 
-    // or multi-line syntax with { ... }, need return here:
+    // ou em sintaxe multi-linha com { ... }, aqui precisa de return
     let sum = (a, b) => {
       // ...
       return a + b;
     }
 
-    // without arguments
-    let sayHi = () => alert("Hello");
+    // sem argumentos
+    let sayHi = () => alert("Olá");
 
-    // with a single argument
+    // com um único argumento
     let double = n => n * 2;
     ```
 
 
-- Functions may have local variables: those declared inside its body. Such variables are only visible inside the function.
-- Parameters can have default values: `function sum(a = 1, b = 2) {...}`.
-- Functions always return something. If there's no `return` statement, then the result is `undefined`.
+- Funções podem ter variáveis locais: aquelas declaradas no seu corpo. Tais variáveis apenas são visíveis dentro da função.
+- Parâmetros podem ter valores por defeito: `function sum(a = 1, b = 2) {...}`.
+- Funções sempre retornam algo. Se não houver uma instrução `return`, então o resultado é `undefined`.
 
 
-| Function Declaration | Function Expression |
+| Declaração de Função | Expressão de Função |
 |----------------------|---------------------|
-| visible in the whole code block | created when the execution reaches it |
-|   - | can have a name, visible only inside the function |
+| visível em todo o bloco de código | criada quando a execução a alcança |
+|   - | pode ter um nome, visível apenas dentro da função |
 
-More: see <info:function-basics>, <info:function-expressions-arrows>.
+Mais: veja <info:function-basics>, <info:function-expressions-arrows>.
 
-## More to come
+## Mais adiante
 
-That was a brief list of JavaScript features. As of now we've studied only basics. Further in the tutorial you'll find more specials and advanced features of JavaScript.
+Esta foi uma breve lista de funcionalidades de JavaScript. Até agora apenas estudámos o básico. Mais adiante no tutorial encontrará mais funcionalidades especiais e avançadas de JavaScript.
```

Comment:
- "estudamos" whitout the accent
- "Mais adiante no tutorial **você encontrará**"